lib/fs.js

```diff
@@ -293,7 +303,12 @@ ReadFileContext.prototype.close = function(err) {
   req.oncomplete = readFileAfterClose;
   req.context = this;
   this.err = err;
-  binding.close(this.fd, req);
+
+  if (this.isUserFd) {
+    req.oncomplete.call(req, null);
```

Comment:
Yes, I agree that this is important information. The current patch adds the phrase

> Specified file descriptors will not be closed automatically.

to all relevant locations. Do you think the whole thing should be bolded, or just the "not"? I am currently leaning towards emphasizing the whole phrase.